ush/forecast_postdet.sh

```diff
@@ -235,21 +235,13 @@ EOF
     iopt_stc=${iopt_stc:-"1"}
   fi
 
-  # Scan suite file to determine whether it uses UGWP v1
-  if [ $(grep -i ugwpv1_gsldrag ${_suite_file} | wc -l ) -gt 0 ]; then
-    gwd_opt="2"
-    knob_ugwp_version="1"
-    OROFIX_ugwd=${OROFIX_ugwd:-"${FIX_DIR}/fix_ugwd"}
-    $NLN ${OROFIX_ugwd}/ugwp_limb_tau.nc $DATA/ugwp_limb_tau.nc
-    for n in $(seq 1 $ntiles); do
-      $NLN ${OROFIX_ugwd}/$CASE/${CASE}_oro_data_ls.tile${n}.nc $DATA/INPUT/oro_data_ls.tile${n}.nc
-      $NLN ${OROFIX_ugwd}/$CASE/${CASE}_oro_data_ss.tile${n}.nc $DATA/INPUT/oro_data_ss.tile${n}.nc
-    done
-  else
-    gwd_opt="1"
-    knob_ugwp_version="0"
-    launch_level=${launch_level:-$(echo "$LEVS/2.35" |bc)}
-  fi
+  # Files for GWD
```

Comment:
Why was auto-detection of UGWP from the suite file removed?

Reply:
The oro_data_*.nc files are needed for v0 or v1.

Reply:
Okay, that part can be moved outside. But the auto-detection of gwd_opt should remain and all the gwd_opt modifications that were made to config.fcst can be here instead.

Reply:
Leave for now